zipkin-lens/testdata/yelp.json

```diff
@@ -0,0 +1,360 @@
+    "remoteEndpoint": {
+      "serviceName": "spectre",
+      "port": 31286
+    },
+    "tags": {
+      "client_status_code": "200",
+      "http.uri.client": "<uri_path>",
+      "request_budget": "9989",
+      "tracer": "syslog2scribe.haproxy-synapse"
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "f5f268651b2a2b34",
+    "id": "7a778764a0d0b594",
+    "kind": "SERVER",
+    "name": "get",
+    "timestamp": 1571896375310740,
+    "duration": 1490,
+    "localEndpoint": {
+      "serviceName": "spectre"
+    },
+    "tags": {
+      "ecosystem": "prod",
+      "habitat": "uswest1aprod",
+      "http.uri.client": "<uri_path>",
+      "region": "uswest1-prod"
+    },
+    "shared": true
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "f5f268651b2a2b34",
+    "id": "6a65182ea4f684c3",
+    "kind": "CLIENT",
+    "name": "set mobile_api_nonce",
+    "timestamp": 1571896375302030,
+    "duration": 1026,
+    "localEndpoint": {
+      "serviceName": "mobile_api",
+      "port": 31049
+    },
+    "remoteEndpoint": {
+      "serviceName": "memcache"
+    },
+    "tags": {
+      "driver": "yelp_memcache",
+      "method": "set",
+      "requests": "1",
+      "system": "mobile_api_nonce",
+      "ttl": ""
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "f5f268651b2a2b34",
+    "id": "cb4d73f31cd90cae",
+    "kind": "CLIENT",
+    "name": "get_multi mobile_api_nonce",
+    "timestamp": 1571896375300642,
+    "duration": 1066,
+    "localEndpoint": {
+      "serviceName": "mobile_api",
+      "port": 31049
+    },
+    "remoteEndpoint": {
+      "serviceName": "memcache"
+    },
+    "tags": {
+      "driver": "yelp_memcache",
+      "hits": "0",
+      "method": "get_multi",
+      "requests": "1",
+      "system": "mobile_api_nonce"
+    }
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "2e8cfb154b59a41f",
+    "id": "f5f268651b2a2b34",
+    "kind": "SERVER",
+    "name": "post /location/update/v4",
+    "timestamp": 1571896375297103,
+    "duration": 41740,
+    "localEndpoint": {
+      "serviceName": "mobile_api",
+      "port": 31049
+    },
+    "tags": {
+      "ecosystem": "prod",
+      "habitat": "uswest1bprod",
+      "http.route": "/location/update/v4",
+      "http.uri": "/location/update/v4",
+      "http.uri.qs": "/location/update/v4",
+      "region": "uswest1-prod",
+      "response_status_code": "200",
+      "version_SHA": "6535284b1699df0a766384a648dc95c462a7313d"
+    },
+    "shared": true
+  },
+  {
+    "traceId": "a03ee8fff1dcd9b9",
+    "parentId": "2e8cfb154b59a41f",
+    "id": "f5f268651b2a2b34",
+    "kind": "CLIENT",
+    "name": "post",
+    "timestamp": 1571896375287000,
+    "duration": 56000,
+    "localEndpoint": {
+      "serviceName": "yelp-main"
+    },
+    "remoteEndpoint": {
+      "serviceName": "mobile_api",
+      "port": 31049
+    },
+    "tags": {
+      "client_status_code": "200",
+      "http.uri.client": "<uri_path>",
```

Comment:
can you replace these with dummy values with similar cardinality? I presume  '<uri_path>' is not what's recorded.